cmd/libs/go2idl/client-gen/test_apis/testgroup/v1/types.go

```diff
@@ -32,6 +33,8 @@ type TestType struct {
 	Status TestTypeStatus `json:"status,omitempty"`
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Comment:
it seems weird that you have to specify this interface.  What else would you generate a deep copy for?